### PR TITLE
RFC005: Ported changes back after implementation

### PR DESCRIPTION
**Pull request by reinkrul**
Impl: https://github.com/nuts-foundation/nuts-node/issues/337

### CODE REVIEW

**Comment by stevenvegt** [comment]
Why is the PeerID removed?
Can you write in the introduction of the document to check chapter 7 for the protobuf spec?

**Comment by reinkrul** [comment]
> Why is the PeerID removed?

Oops, that wasn't intended...

File: .gitbook/assets/rfc005/network.proto [hunk 1]
```diff
@@ -103,23 +102,18 @@ message TransactionPayload {
...
     repeated string peers = 3;
     // numberOfTransactions contains the total number of transactions on the node's DAG.
     uint32 numberOfTransactions = 4;
-    // version contains an indication of the software version of the node. It's recommended to use a (Git) commit ID that uniquely resolves to a code revision, alternatively a semantic version could be used (e.g. 1.2.5).
-    string version = 10;
-    // vendor contains an indication of the vendor of the software of the node. For open source implementations it's recommended to specify URL to the public, open source repository.
-    // Proprietary implementations could specify the product's or vendor's name.
-    string vendor = 11;
+    // softwareVersion contains an indication of the software version of the node. It's recommended to use a (Git) commit ID that uniquely resolves to a code revision, alternatively a semantic version could be used (e.g. 1.2.5).
+    string softwareVersion = 10;
```
Comment (woutslakhorst) [inline .gitbook/assets/rfc005/network.proto:114]: in master it's still called `version`

File: .gitbook/assets/rfc005/network.proto [hunk 2]
```diff
@@ -103,23 +102,18 @@ message TransactionPayload {
...
+    // softwareID contains an identification of the particular Nuts implementation of the node.
+    // For open source implementations it's recommended to specify URL to the public, open source repository.
+    // Proprietary implementations could specify the product or vendor's name.
+    string softwareID = 11;
```
Comment (woutslakhorst) [inline .gitbook/assets/rfc005/network.proto:118]: in master it's still called `vendor`

**Review by woutslakhorst — CHANGES_REQUESTED** [review]
and indeed what happened to PeerID?

File: .gitbook/assets/rfc005/network.proto (hunk 1 above)
Comment (reinkrul) [reply .gitbook/assets/rfc005/network.proto:114]: This changed after discussion with @stevenvegt and @gerardsn, will be changed in `nuts-node` (same for softwareID).

**Review by woutslakhorst — APPROVED** [review]
Please link the issue for changing the version and vendor fields

**Pull request merged**